### DISCUSSION TITLE
android 10 bootstraps & package build scripts

### DISCUSSION THREAD

**Discussion by simbadMarino** [termux/termux-app#2253] [General]
Hello there @agnostic-apollo @xeffyr , its me again! So I've finally made it and have a sort-of stable version of my app(https://github.com/simbadMarino/dCloud) in which I forked termux and used a bit of ReactNative to call it, however I did it for SDK28 as it was the easiest way to make the MVP a that time... Now I'm planning to migrate the whole termux part to SDK29 so I can publish it on Google Play, however I see that the android 10 branch is a bit outdated and could not find any info related to bootstraps files or build scripts..
Would you please help by providing the old bootstraps that were located at bintray to test the branch? and if possible similar build scripts as the ones available for SDK28? I can later on support the termux project with those bootstraps repositories if needed ;)

Thanks in advance!

**Comment by agnostic-apollo** [comment]
The `android-10` branch is very outdated, a newer one was [`updated-flavor`](https://github.com/termux/termux-app/tree/updated-flavor) but haven't been tested by me and is missing stuff. For generating the bootstrap for `targetSdkVersion>=29`, you can pass the `--android10` flag to [`generate-bootstraps.sh`](https://github.com/termux/termux-packages/blob/master/scripts/generate-bootstraps.sh), it should probably work. My custom `build-bootstraps.sh` also supports it but I haven't tested it. The `bintray` one would be outdated and not of much use, I don't think I have the `bintray` one, I deleted it probably.
